rest-api/src/main/scala/za/co/absa/enceladus/rest_api/HDFSConfig.scala

```diff
@@ -42,22 +41,25 @@ class HDFSConfig @Autowired() (spark: SparkSession) {
   val krb5username: String = ""
   @Value("${menas.hadoop.auth.krb5.keytab:}")
   val krb5keytab: String = ""
+  @Value("${menas.hadoop.conf.dir:}") // TODO change to rest-api
```

Comment:
Do you perhaps want to reference an issue here? E.g. #2111

Reply:
I want to wait for @jakipatryk to merge his rename first.

Reply:
Done